cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs

```diff
@@ -1151,6 +1160,78 @@ pGovernanceCmd =
                         <*> pProtocolParametersUpdate
                         <*> optional pCostModels
 
+    pGovernanceCreatePoll :: Parser GovernanceCmd
+    pGovernanceCreatePoll =
+      GovernanceCreatePoll
+        <$> pPollQuestion
+        <*> some pPollAnswer
+        <*> optional pPollNonce
+        <*> pOutputFile
+
+    pGovernanceAnswerPoll :: Parser GovernanceCmd
+    pGovernanceAnswerPoll =
+      GovernanceAnswerPoll
+        <$> pPollFile
+        <*> pSigningKeyFile Input
+        <*> optional pPollAnswerIndex
```

Comment:
I know you disagree with having a file as an output, however we should maintain consistency in `cardano-cli` and include this option.

Reply:
Feel free to do it.

Reply:
No, when you are contributing to a project, take the time to contribute code that is consistent with their style. "Feel free to do it" is unacceptable and disrespectful.